core/controllers/question_editor.py

```diff
@@ -29,14 +29,30 @@
 class QuestionCreationHandler(base.BaseHandler):
     """A handler that creates the question model given a question dict."""
 
+    def validate_skill_ids(self, skill_ids):
+        """Validates that the passed skill ids are a list of strings."""
+        if not(isinstance(skill_ids, basestring)) and not(
+                isinstance(skill_ids, list) and all(
+                    [isinstance(
+                        skill_id, basestring) for skill_id in skill_ids])):
+            raise self.InvalidInputException
+
     @acl_decorators.can_manage_question_skill_status
-    def post(self, skill_id):
+    def post(self, skill_ids):
         """Handles POST requests."""
-        skill_domain.Skill.require_valid_skill_id(skill_id)
-        skill = skill_services.get_skill_by_id(skill_id, strict=False)
-        if skill is None:
+        try:
+            skill_ids = skill_ids.split(',')
```

Comment:
Avoid redefining vars, use comma_separated_skill_ids for the controller arg.

Reply:
Done